src/constants.ts

```diff
@@ -15,9 +15,9 @@ export const ALIASING_LIMIT = 2;
 /* "magic number" for bezier approximations of arcs (http://itc.ktu.lt/itc354/Riskus354.pdf) */
 export const kRect = 1 - 0.5522847498;
 
-export const LEFT_CLICK = 1;
-export const MIDDLE_CLICK = 2;
-export const RIGHT_CLICK = 3;
+export const LEFT_CLICK = 0;
+export const MIDDLE_CLICK = 1;
+export const RIGHT_CLICK = 2;
```

Comment:
we can remove these values or not, they are used in a single place now
now they are the same values as `MouseEvent#button`